README.md

```diff
@@ -1,18 +1,51 @@
-Hosted website Link https://guru-jii.herokuapp.com/
+## Technology Stack used:
+<img src="https://img.shields.io/badge/reactjs%20-61DAFB.svg?&style=for-the-badge&logo=react&logoColor=blue"/> <img src="https://img.shields.io/badge/github%20-%23121011.svg?&style=for-the-badge&logo=github&logoColor=white"/> <img src="https://img.shields.io/badge/heroku%20-%23430098.svg?&style=for-the-badge&logo=heroku&logoColor=white"/> <img src="https://img.shields.io/badge/express.js%20-%23404d59.svg?&style=for-the-badge"/> <img src ="https://img.shields.io/badge/MongoDB-%234ea94b.svg?&style=for-the-badge&logo=mongodb&logoColor=white"/>
 
-Want to contribute? Follow the steps below to set it up locally.
+- **Frontend**: ReactJS
+- **Backend**: Node.js, Express.js
+- **IDE**: VS Code
+- **Version Control**: Git and GitHub
+- **Database**: MongoDb
+- **Hosting**: Heroku
 
-Step 1: Fork and Clone the repo.<br />
-Step 2: run `cd backend yarn install` to install backend dependencies.<br />
-step 3: run `cd .. yarn install` to install frontend dependencies.<br />
-Step 4: run `yarn dev`. This should launch the website on `localhost:3000`.
+## Getting Started 🚀 
+### Want to contribute? Follow the steps below to set it up locally.
+
+```
+# Clone this repository
+$ git clone https://github.com/your-username/MERN-Gurujii-dev.git
+
+# Go into the repository
+$ cd MERN-Gurujii-dev
+
+# Install backend dependencies
+$ cd backend yarn install
+
+# Install frontend dependencies
+$ cd .. yarn install
+
+# Run website
+$ yarn dev
+```
+This should launch the website on `localhost:3000`.
 
```

Comment:
It would be nice to add some notes as few contributors are confused and making the same mistake.

- This project uses yarn to install the dependencies. Don't use `npm install`.
- For security reasons I have made the MongoDb [URI](https://github.com/Namanl2001/MERN-Gurujii-dev/blob/b723dcee3e07e4a9c62ddb7bf59a2193f5390961/backend/server.js#L22) secure, by not adding a hardcoded username and password. For development purpose, you can install, MongoDB locally or use the online version.